app/controllers/users/details.js

```diff
@@ -47,7 +53,42 @@ export default Ember.Controller.extend(OrganisationMixin, {
     }
   ),
 
+  disableUserAccount(disabled) {
```

Comment:
If it disabling user, there is no need to pass false `params` . You can directly set `false` or If you are thinking for of using this function for both and enabling user then "ToggleUserAccount" make will make more sense i think.

Reply:
I think `disableUserAccount` sounds more like an action, rather than a property. I think it would be good to move the whole thing into `disableUser()` action written below